src/common/libschedutil/init.h

```diff
@@ -29,6 +29,7 @@ typedef struct schedutil_ctx schedutil_t;
  */
 schedutil_t *schedutil_create (flux_t *h,
                                const struct schedutil_ops *ops,
+                               int flags,
                                void *arg);
```

Comment:
This may be entirely personal preference so feel free to ignore, but generally in our API, callback functions and their opaque arguments are next to each other in parameter lists.  This is a struct of callbacks, so  not quite the same thing.

The suggestion would be to move `flags` between `h` and `ops` for consistency, but feel free to ignore if you don't see it.